cvm-securekey-release-app/Main.cpp

```diff
@@ -39,10 +39,24 @@ enum class Operation
     Undefined
 };
 
+// Check if tracing is to be enabled for SKR in the env.
+void set_tracing(void)
+{
+    auto skr_trace_flag = std::getenv("SKR_TRACE_ON");
+    if(skr_trace_flag != nullptr && strlen(skr_trace_flag) > 0)
+    {
+        if(strcmp(skr_trace_flag, "1") ==0)

```

Comment:
As we discussed in our 1-1s, pls implement additional level of tracing. Level 2 will be more verbose than level 1.

Reply:
let me know if you want to take this PR as it is with 1 level only; and follow up with another PR for level 2.

Reply:
Yes, I am implementing it, I will do both level 1 and 2 in same PR.